src/libasr/pass/intrinsic_function_registry.h

```diff
@@ -727,9 +810,13 @@ namespace Abs {
             ASR::symbol_t *s = scope->get_symbol(func_name);
             ASR::Function_t *f = ASR::down_cast<ASR::Function_t>(s);
             return b.Call(s, new_args, expr_type(f->m_return_var),
-                compile_time_value);
+                                compile_time_value);
         }
+
+        func_name = scope->get_unique_name(func_name);
+
```

Comment:
```suggestion
```
get_unique_name is present in the `declare_basic_variables` macro.

Reply:
I can help you with the `intrinsic_function_registry` and I also apologize for not submitting the sync PR with these changes to LFortran, I will submit it soon.

Reply:
On my system everything worked with LPython after the sync. I am yet to test it with LFortran.